templates/ra_guac_private_autoscale_elb.element

```diff
@@ -449,7 +460,15 @@
             },
             "Properties" :
             {
-                "ImageId": { "Fn::GetAtt": [ "AmiIdLookup", "Id" ] },
+                "ImageId" :
+                {
+                    "Fn::If" :
+                    [
+                        "UseAmiLookup",
+                            {"Fn::GetAtt" : [ "AmiIdLookup", "Id" ]},
+                            {"Ref" : "AmiId"}
```

Comment:
Too much indent on these two lines. They should both align with `"UseAmiLookup".